develop/devguide/Automation/UIBlockTypesOverview.md

```diff
@@ -270,6 +270,11 @@ All files uploaded by users will by default be placed in the *C:\\Skyline DataMi
 > [!TIP]
 > See also: [GetUploadedFilePath](xref:Skyline.DataMiner.Automation.UIResults#Skyline_DataMiner_Automation_UIResults_GetUploadedFilePath_System_String_)
 
+> [!NOTE]
+> Contrary to the other UI block types the FileSelector doesn't allow to set an initial value. During an automation script session, it is however possible (since RN37620) to keep the files that were already uploaded after the UI was shown.
+> In a dashboard or low-code app, the UI block needs to keep the same row, column and DestVar within the session. If a block of a different type or DestVar is at same location or that location has no blocks when the UI is shown again, the information about the uploaded files is lost.
+> In Cube the UI block needs to keep the same DestVar within the session. If the file selector block with the same DestVar is missing whe the UI is shown again, the information about the uploaded files is lost.
```

Comment:
needs to keep the same DestVar within the session ->  needs to retain the DestVar within the same session